Core/InitializeChemistry.m

```diff
@@ -131,7 +131,13 @@
 
 Gstr = Gstr(1:i,:);
 iempty = cellfun('isempty',Gstr);
-Gstr(iempty) = {'ONE'}; %replace empty cells with name of "ONE" species (all 1's)
+Gstr(iempty) = {'ONE'}; %replace empty cells with name of "ONE" species (all 1's)\
+%%%%% Add in code to deal with limiting reactants
```

Comment:
Probably cleaner to deal with these special "limiting reagent" reactions by adding a boolean flag that can be thrown for individual reactions within chemistry scripts. Then the Gstr values in reactions would have normal names and this bit of code could be deleted.